internal/node/node.bzl

```diff
@@ -277,7 +277,7 @@ _NODEJS_EXECUTABLE_ATTRS = {
     "data": attr.label_list(
         doc = """Runtime dependencies which may be loaded during execution.""",
         allow_files = True,
-        aspects = [node_modules_aspect, module_mappings_runtime_aspect],
+        aspects = [node_modules_aspect, module_mappings_aspect, module_mappings_runtime_aspect],
```

Comment:
I was hoping to remove the module_mappings_runtime_aspect at the same time. This is a perf regression since we'll walk the deps two separate times, gathering the same data and returning it in slightly different format.
I guess we can do that as follow-up but ideally we should do it now? Soon?